tools/addrundata/case.go

```diff
@@ -31,18 +31,13 @@ func (tc *testcase) read(testdir string) error {
 	if err != nil {
 		return fmt.Errorf("could not glob: %w", err)
 	}
-	for _, testpath := range testpaths {
-		if err := readFile(testpath, tc.readPackage); err != nil {
-			return fmt.Errorf("could not detect test package: %w", err)
-		}
-		if tc.pkg != "" {
-			break
-		}
+	if len(testpaths) == 0 {
```

Comment:
Previously, we allow a test package to have no code and just README.md, and rundata_test.go would be generated with an assumed package name that is taken from the basename of the testdir.

Now it would be impossible to generate a rundata_test.go for a new test without at least some dummy code, which seems like a regression.

Revert all the changes in this function? Feel free to add a comment about assumed package name.

Reply:
Where is the logic to infer the package name?

Reply:
behavior reverted